sonaranalyzer-dotnet/src/SonarAnalyzer.Utilities/Rules.Description/S1479_cs.html

```diff
@@ -1,5 +1,61 @@
-<p>When <code>switch</code> statements have large sets of <code>case</code> clauses, it is usually an attempt to map two sets of data. A real map
-structure would be more readable and maintainable, and should be used instead.</p>
+<p>When <code>switch</code> statements have large sets of case clauses, it is usually an attempt to map two sets of data. A <code>Dictionary</code>
+should be used instead to make the code more readable and maintainable.</p>
+<h2>Noncompliant Code Example</h2>
+<p>With a “Maximum number of case” set to 14</p>
+<pre>
+public class TooManyCase
+{
+    public int switchCase(char ch)
+    {
+        switch(ch) {  // Noncompliant
+            case 'a':
+            case 'b':
+            case 'c':
+            case 'd':
+            case 'e':
+                return 1;
+            case 'f':
+            case 'g':
+            case 'h':
+            case 'i':
+            case 'j':
+                return 2;
+            case 'k':
+            case 'l':
+            case 'm':
+            case 'n':
+            case 'o':
+                return 3;
+            default:
+                return 4;
```

Comment:
This noncompliant example is actually wrong, as the rule has an exception that "fall-through" cases are ignored, this would actually not raise an issue